packages/deprecation-crawler/src/lib/tasks/ensure-deprecations-url-config.ts

```diff
@@ -20,3 +20,15 @@ export async function ensureDeprecationUrlConfig(
     ...userConfig,
   };
 }
+
+export function getSuggestionsFormPackageJson(config: CrawlConfig): string {
+  const pkg = getSiblingPgkJson(config.tsConfigPath);
+  let url = '';
+  if (pkg.homepage) {
+    url = pkg.homepage;
+  } else if (pkg.repository?.url) {
+    url = pkg.repository.url;
+  }
+
+  return url + '/deprecations';
```

Comment:
Should we do something if `homepage` or `repository.url` isn't in the package.json?